sjcl/build.boot

```diff
@@ -0,0 +1,40 @@
+(set-env!
+  :resource-paths #{"resources"}
+  :dependencies '[[cljsjs/boot-cljsjs "0.5.2" :scope "test"]])
+
+(require '[boot.task-helpers]
+         '[cljsjs.boot-cljsjs.packaging :refer :all])
+
+
+(def +lib-version+ "1.0.6")
+(def +version+ (str +lib-version+ "-1"))
+
+(task-options!
+  push {:ensure-clean false}
+  pom  {:project     'cljsjs/sjcl
+        :version     +version+
+        :description "Stanford Javascript Crypto Library"
+        :url         "https://github.com/bitwiseshiftleft/sjcl/blob/master/sjcl.js"
+        :license     {"BSD" "https://opensource.org/licenses/BSD-3-Clause"}
+        :scm         {:url "https://github.com/cljsjs/packages"}})
+
+(require '[boot.core :as c]
```

Comment:
You're requiring a bunch of stuff here, but as far as I can tell, none of them are used anywhere.